### PR TITLE
Note about PHP 7.4 support

### PR DESCRIPTION
**Pull request by konradoboza**
| Question      | Answer
| ------------- | ---
| Versions      | `2.5`, `3.0`, `master`

### CODE REVIEW

File: docs/getting_started/requirements.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ For security and performance we generally recommend (unless otherwise noted and
     -   7.1
     -   7.2
     -   7.3
+    -   7.4 (does not apply to eZ Platform Cloud)
```
Comment (andrerom) [inline docs/getting_started/requirements.md:54]: Maybe something like this @DominikaK ?

```suggestion
    -   7.4 _(NOTE: Support on eZ Platform Cloud/Platform.sh is currently considered a "preview" as it requires you to [manually installing php-redis](https://docs.platform.sh/languages/php/redis.html#using-the-redis-builder-script) yourself)_
```

Comment (DominikaK) [inline docs/getting_started/requirements.md:54]:
```suggestion
    -   7.4 (support on eZ Platform Cloud/Platform.sh is currently considered a "preview" as it requires you to [manually install php-redis](https://docs.platform.sh/languages/php/redis.html#using-the-redis-builder-script))
```

Comment (konradoboza) [reply docs/getting_started/requirements.md:54]: +1 from my side.

**Comment by andrerom** [comment]
Had a talk with @lserwatka, and agreed to hold this back until we have full support on p.sh.

Background is change to cloud first approach on product, we won't support features not available on our cloud. We will of course make sure to work with them to soon add support for things we need for full PHP 7.4 support though.

@konradoboza This means from support perspective PHP 7.4 continues to be considered experimental as per the Requirements page, and while we can't give ETA we expect it to be supported soon given progress on code and testing side recently.

**Comment by konradoboza** [comment]
That's fine with me.

**Comment by andrerom** [comment]
@konradoboza They now say that PHP-redis is included: https://docs.platform.sh/languages/php/extensions.html

So I think we can merge this but I'm thinking we might even remove the extra note on redis now if we can simply verify this. I'll pong you in private on how we can verify

**Comment by konradoboza** [comment]
Verified, we are good to proceed. ;)

**Review by DominikaK — APPROVED** [review]

**Review by andrerom — APPROVED** [review]

**Pull request merged**